cloudwatch.tf

```diff
@@ -0,0 +1,114 @@
+resource "aws_cloudwatch_metric_alarm" "Rabbitmq1CPU" {
```

Comment:
Will these names clash per environment? I'm wondering whether each deploy environment should have their own cloudwatch alarms, in which case we should probably be using the env variable somewhere in here?